client/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
what is this for?

Reply:
when i tried using MUI for the syling of the ui it forced me to bypass some of the warnings, difference of versions warnings between MUI and React. I havent committed all of the stylings yet, just wanted to have everything including the backend so i can get started with the algorithm